db/migrate/20220913195914_add_user_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] Same here, Please edit this migration because now it's creating a foreign key as `user_id` and It's supposed to be `author_id` like the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).

Reply:
@Omar-Muhamad , I have created just like this -  rails g migration AddAuthorRefToComments author:references - ERROR:  relation "authors" does not exist.
Do you have any suggestions to fix it?

Reply:
> * [ ]  Same here, Please edit this migration because now it's creating a foreign key as `user_id` and It's supposed to be `author_id` like the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).

- [ ] Same here, Please address this 👍🏻

Reply:
Let's change migration file !!!